src/v/compat/metadata_dissemination_compat.h

```diff
@@ -127,8 +127,8 @@ struct compat_check<cluster::get_leadership_request> {
     static void to_json(
       [[maybe_unused]] cluster::get_leadership_request obj,
       [[maybe_unused]] json::Writer<json::StringBuffer>& wr) {}
-    static cluster::get_leadership_request from_json([
-      [maybe_unused]] json::Value& rd) {
+    static cluster::get_leadership_request
+    from_json([[maybe_unused]] json::Value& rd) {
```

Comment:
looks like this hunk can be squashed into a previosu commit?

Reply:
same...done..